src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsFloorAdhesionSkinContext.java

```diff
@@ -1,6 +1,10 @@
 package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.helpers;
 
+import com.wikia.webdriver.common.core.WikiaWebDriver;
 import com.wikia.webdriver.common.core.helpers.Browser;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.helpers.Browser'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)